packages/web-api/src/WebClient.spec.js

```diff
@@ -425,7 +426,7 @@ describe('WebClient', function () {
     it('should properly serialize simple API arguments', function () {
       const scope = nock('https://slack.com')
         // NOTE: this could create false negatives if the serialization order changes (it shouldn't matter)
-        .post(/api/, 'token=xoxb-faketoken&team_id=T12345678&foo=stringval&bar=42&baz=false')
+        .post(/api/, 'team_id=T12345678&foo=stringval&bar=42&baz=false')
         .reply(200, { ok: true });
       return this.client.apiCall('method', { foo: 'stringval', bar: 42, baz: false, team_id: 'T12345678' })
```

Comment:
Can we override the token in the authorization header if a developer passes a token like `this.client.apiCall('method', { token: 'xoxb-' })`? Switching tokens is not a rare use case. For instance, the singleton `WebClient` does not hold any token and your app utilizes multiple user tokens for user related API calls.